.github/CODEOWNERS

```diff
@@ -8,7 +8,12 @@
 
 # DevOps for Actions and other workflow changes.
 .github/workflows @bitwarden/dept-devops
-util/MsSqlMigratorUtility/Dockerfile @bitwarden/dept-devops
+
+# DevOps for Docker changes.
+**/Dockerfile @bitwarden/dept-devops
+**/*.Dockerfile @bitwarden/dept-devops
+**/.dockerignore @bitwarden/dept-devops
+**/prepare-env.sh @bitwarden/dept-devops
```

Comment:
❓ This file doesn't seem to exist anywhere -- should it be excluded?

Reply:
I thought adding it for the future. But removed it as you suggest.